File: network/src/main/java/pixel/kotlin/bassblog/network/PostsResponse.java
package pixel.kotlin.bassblog.network;


import java.util.List;

public class PostsResponse {
    private String nextPageToken;
    private List<RawPost> items;

    public String getNextPageToken() {
        return nextPageToken;
    }

    public List<RawPost> getItems() {
        return items;
    }

    public class RawPost {
        public String id;
        public String title;
        public String content;
        public String published;
        public List<String> labels;
        public List<RawImages> images;

    }

    public class RawImages {
        public String url;
    }
}

File: app/src/main/java/pixel/kotlin/bassblog/network/PostsResponse.kt
package pixel.kotlin.bassblog.network

class PostsResponse {
    val nextPageToken: String? = null
    val items: List<RawPost>? = null

    inner class RawPost {
        var id: Long = 0L
        var title: String? = null
        var url: String? = null
        var content: String? = null
        var published: String? = null
        var labels: List<String>? = null
        var images: List<RawImages>? = null
    }

    inner class RawImages {
        var url: String? = null
    }
}


